src/Sulu/Bundle/ContactBundle/Tests/Functional/Controller/AccountControllerTest.php

```diff
@@ -1256,12 +1256,12 @@ public function testPutNoDetails()
 
         $this->assertEquals('ExampleCompany', $response->name);
 
-        $this->assertEquals(0, sizeof($response->urls));
-        $this->assertEquals(0, sizeof($response->emails));
-        $this->assertEquals(0, sizeof($response->phones));
-        $this->assertEquals(0, sizeof($response->faxes));
-        $this->assertEquals(0, sizeof($response->notes));
-        $this->assertEquals(0, sizeof($response->addresses));
+        $this->assertEquals(0, count($response->urls));
```

Comment:
Actually there is even a fixer doing that... But it is marked that it might change code behavior, that's why I don't really want to use it...